src/lib.rs

```diff
@@ -24,11 +24,14 @@ pub mod console {
 pub mod gpio {
     use libtock_gpio as gpio;
     pub type Gpio = gpio::Gpio<super::runtime::TockSyscalls>;
-    pub use gpio::{
-        Error, GpioInterruptListener, GpioState, InputPin, OutputPin, PinInterruptEdge, Pull,
-        PullDown, PullNone, PullUp,
-    };
+    pub use gpio::{Error, GpioInterruptListener, GpioState, InputPin, OutputPin, PinInterruptEdge, Pull, PullDown, PullNone, PullUp};
```

Comment:
```suggestion
    pub use gpio::{
        Error, GpioInterruptListener, GpioState, InputPin, OutputPin, PinInterruptEdge, Pull,
        PullDown, PullNone, PullUp,
    };
```
I ran `cargo fmt` and this is the formatted source.